docs/docs-build-requirements.txt

```diff
@@ -7,6 +7,7 @@
 docutils==0.15.2
 Sphinx==3.5.2
 sphinx-click==2.5.0
+sphinx-toolbox==2.5.0
```

Comment:
is there a reason why we need to pin this dependency? i patched this pr and tried the following:
```
diff --git a/docs/docs-build-requirements.txt b/docs/docs-build-requirements.txt
index 9127abe2e..5628ecb35 100644
--- a/docs/docs-build-requirements.txt
+++ b/docs/docs-build-requirements.txt
@@ -4,10 +4,9 @@
 -e ../python_modules/dagster-graphql
 -e ../python_modules/libraries/dagster-celery
 -e ../python_modules/dagit
-docutils==0.15.2
 Sphinx==3.5.2
 sphinx-click==2.5.0
-sphinx-toolbox==2.5.0
+sphinx-toolbox
 recommonmark==0.7.1
 nbsphinx==0.8.2
 pendulum==1.4.4
```
and the sphinx build test turned green locally

Reply:
nice -- zero thought went into the pin, so I'm happy to remove it